packages/core/src/artifact/Artifact.ts

```diff
@@ -0,0 +1,17 @@
+export interface Artifact {
```

Comment:
jeśli chodzi o typy to zazwyczaj robimy tak że:
1. Jak projekt jest mały to w src/ jest jeden plik types.ts i tam są wszystkie typy/interfejsy
2. Jak projekt jest większy to w scr/ robimy katalog types/ i tam wrzucamy osobne pliki tak jak Ty zrobiłeś